diktat-rules/src/test/resources/test/smoke/src/main/kotlin/Example4Expected.kt

```diff
@@ -1,45 +1,10 @@
 package org.cqfn.diktat
 
-class SpecialTagsInKdoc {
```

Comment:
Why is everything else removed in this example?

Reply:
Was testing, forgot to return it